### PR TITLE
add travis to repo

### PR DESCRIPTION
**Pull request by kneemer**
Fixed issues causing failures on travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: node_js
+before_install: npm install -g semistandard
```
Comment (jgravois) [inline .travis.yml]: isn't it sufficient that [`semistandard`](https://github.com/Esri/esri-leaflet-renderers/blob/master/package.json#L49) is included as a devDependency of the project?

Comment (kneemer) [inline .travis.yml]: possibly, I just know it always suggest that it should be global when npm installing.  Should I take it out and see what happens?

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   "dependencies": {
      "esri-leaflet": "^2.0.0-beta.5",
      "leaflet": "^1.0.0-beta.1",
-     "leaflet-shape-markers": "^1.0.3"
+     "leaflet-shape-markers": "^1.0.4"
```
Comment (jgravois) [inline package.json]: nothing wrong with bumping this, but even `^1.0.0` will ensure that the newest thing (up to 2.0.0) is installed (more info [here](http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json)).

Comment (kneemer) [inline package.json]: I just wanted to make it explicit just in case someone (unlikely) had pulled down this repo and installed an earlier version.

Comment (jgravois) [inline package.json]: sounds good to me.

**Pull request merged**